domains/transfer/Transfer.API/API.IntegrationTests/Controllers/ConnectionInvitationsControllerTests.cs

```diff
@@ -46,4 +43,78 @@ public async Task CreateConnectionInvitation_ShouldReturnUnauthorized_WhenUnauth
 
         result.StatusCode.Should().Be(HttpStatusCode.Unauthorized);
     }
+
+    [Fact]
+    public async Task GetConnectionInvitation_ShouldReturnOK_WhenInvitationExists()
```

Comment:
Hvad blev der af den test, der oprettede via POST og derefter brugte GET til at verificere oprettelse?

Reply:
Fixed in https://github.com/Energinet-DataHub/energy-origin/pull/415/commits/f040ef535680294e0a47ae5624b620361a4cebcf